cmake/OpenCVFindLibsGrfmt.cmake

```diff
@@ -153,8 +153,20 @@ if(NOT WEBP_VERSION AND WEBP_INCLUDE_DIR)
   endif()
 endif()
 
+# --- libopenjp2 (optional, check before libjasper) ---
+if(WITH_OPENJPEG)
+  find_package(OpenJPEG QUIET)
+
+  if(OpenJPEG_FOUND)
+    message(STATUS "Found OpenJPEG: ${OPENJPEG_LIBRARIES} (found version \"${OPENJPEG_MAJOR_VERSION}.${OPENJPEG_MINOR_VERSION}.${OPENJPEG_BUILD_VERSION}\")")
+    set(HAVE_OPENJPEG YES)
+  else()
+    message(STATUS "Could NOT find OpenJPEG")
```

Comment:
> find_package(OpenJPEG QUIET)
> message(...)
> message(...)

What if we remove QUIET and following `message()` lines? What is the difference?

Reply:
The QUIET actually has no effect, OpenJPEGConfig.cmake has no message()s at all.

Reply:
Removed the "QUIET"

Reply:
Please return "QUIET" and messages back (OpenJPEG is not a regular CMake module, so large warning are showed if it is not available)